project.clj

```diff
@@ -92,6 +104,7 @@
                          [puppetlabs/jdbc-util "1.0.0"]
                          [puppetlabs/typesafe-config "0.1.5"]
                          [puppetlabs/ssl-utils "0.8.3"]
+                         [puppetlabs/jruby-utils "0.8.0"]
```

Comment:
The reason I was getting jruby-utils conflicts was two different versions of puppetserver, one as a transitive dep of the PE extensions, the other as a dev dependency. Updating the dev dependency to match fixed it. I'm not sure why this pin is here.